CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Added `anonymize_df` task function to `task_utils.py` to anonymize data in the dataframe in selected columns.
+- Added `GET` connection inside the method `genesys_api_connection`.
+
+### Changed
+- Changed the method to call Genesys API and its name, providing a new paremeter `end_point` to be more general.
+Name changed from `genesys_generate_exports` to `genesys_api_connection`.
```

Comment:
Please rephrase to be more clear, below is the example:
Genesys API call method and the name changed from `genesys_generate_exports` to `genesys_api_connection`. Introduced a new `end_point` parameter to make it more generic.

Reply:
I'm not very good at redacting sentences, so I agree with your proposal.